### ISSUE TITLE
Create Blazor version

### ISSUE THREAD

**Issue by KirisameMarisa** [KirisameMarisa/BocchiTracker#12]

**Comment by KirisameMarisa** [comment]
I'm busy with the "Create Plugin (#13)" issue so  thank you for the help!
I'm thinking of making a Desktop Blazor but I've never made a Blazor before, so if you have a better way to make one, please let me know.

**Comment by KirisameMarisa** [comment]
I'm so sorry... The config tool was suddenly asking for the auth yaml file...
I fixed (#19) that the application can run even if the auth file is not found.

And I aggred you make Web Blazor! Great!

**Comment by KirisameMarisa** [comment]
I finally released version v1.0.0! and I'm really sorry for updating destructive...
after this, I will plan to add features without making too many disruptive changes 🐱 
I think you should be able to do it without worry @Andi-Hincu

**Comment by Andi-Hincu** [comment]
Update: I started working on this, but had some bad timing so i had to take a break, I will try to upload in a couple days (even if it's not going to be fully working), just so you can take a look at it and maybe check if something needs to be changed.

**Comment by KirisameMarisa** [comment]
Don't worry, I want you to go at your own pace, really helpful for me! 

Okay! I'm looking forward to upload!

I'm working at next version, I added traking UI but would you like to make?
if you will continue to contribute, would you like to join the collaborator?

**Comment by Andi-Hincu** [comment]
I was working on this, and had to clone the project again, after that I'm having trouble with some icons and some missing text, maybe I forgot to set up something in the config, if not, should i open an issue?
![imagen](https://github.com/KirisameMarisa/BocchiTracker/assets/98480588/eaf58957-c050-4244-a2ac-273fa4defbf3)

Also seeing a couple errors in the console, but after looking at the code, this seems to be the intended behavior when a service is not configured:
`BocchiTracker.Client Error: 0 : Cannt get JIRAClient
BocchiTracker.Client Error: 0 : Cannt get RedmineClient
BocchiTracker.Client Error: 0 : Cannt get SlackClient
BocchiTracker.Client Error: 0 : Cannt get GlitlabClient
BocchiTracker.Client Error: 0 : Cannt get DiscordClient
BocchiTracker.Client Error: 0 : Cannt get ExplorerClient
BocchiTracker.Client Error: 0 : Cannt get DropboxClient
BocchiTracker.Client Error: 0 : Cannt get JIRAClient
BocchiTracker.Client Error: 0 : Cannt get RedmineClient
BocchiTracker.Client Error: 0 : Cannt get SlackClient
BocchiTracker.Client Error: 0 : Cannt get GlitlabClient
BocchiTracker.Client Error: 0 : Cannt get DiscordClient
BocchiTracker.Client Error: 0 : Cannt get ExplorerClient
BocchiTracker.Client Error: 0 : Cannt get DropboxClient
BocchiTracker.Client Error: 0 : Cannt get JIRAClient
BocchiTracker.Client Error: 0 : Cannt get RedmineClient
BocchiTracker.Client Error: 0 : Cannt get SlackClient
BocchiTracker.Client Error: 0 : Cannt get GlitlabClient
BocchiTracker.Client Error: 0 : Cannt get DiscordClient
BocchiTracker.Client Error: 0 : Cannt get ExplorerClient
BocchiTracker.Client Error: 0 : Cannt get DropboxClient
BocchiTracker.Client Error: 0 : Cannt get JIRAClient
BocchiTracker.Client Error: 0 : Cannt get RedmineClient
BocchiTracker.Client Error: 0 : Cannt get SlackClient
BocchiTracker.Client Error: 0 : Cannt get GlitlabClient
BocchiTracker.Client Error: 0 : Cannt get DiscordClient
BocchiTracker.Client Error: 0 : Cannt get ExplorerClient
BocchiTracker.Client Error: 0 : Cannt get DropboxClient
BocchiTracker.Client Error: 0 : Cannt get JIRAClient
BocchiTracker.Client Error: 0 : Cannt get RedmineClient
BocchiTracker.Client Error: 0 : Cannt get SlackClient
BocchiTracker.Client Error: 0 : Cannt get GlitlabClient
BocchiTracker.Client Error: 0 : Cannt get DiscordClient
BocchiTracker.Client Error: 0 : Cannt get ExplorerClient
BocchiTracker.Client Error: 0 : Cannt get DropboxClient`

**Comment by KirisameMarisa** [comment]
are these icons? it sure looks like there are no icons... I tried that on the local but I couldn't reproduce...
uum... have you tried to remake ProjectConfig by using BocchiTracker.Config.exe?
if you try that and problem is not solved, please open issue. sorry

![2023-09-07_10h22_11](https://github.com/KirisameMarisa/BocchiTracker/assets/5306954/34271f77-a09a-46fe-9c0d-3e8c19391cc7)

**Comment by Andi-Hincu** [comment]
I've tried that, also tied a couple more things, nothing works, could you share some of your specs? Like windows version, SDK version, and maybe the .Client properties? I tried to run the project in another machine and similar result.

**Comment by KirisameMarisa** [comment]
I tried that again with Windows11(DesktopPC), Windows10(Surface) but couldn't figure it out...
Please create the issue
* I want to know the operating environment.
* I want to know the details of the problem and how to reproduce it.

Major updates to the release:
* left top gear icon ran BocchiTrackerClient.Config.exe but Changed to show UserConfigEdit window
* Existing combo box with keystrokes (Assigne, Label, Watcher) does not work well so I repleased with its own
    * > Dropdown menus appear at the top left corner on the screen
    * (↑ Perhaps you mean this one?) like you say comobo box is shown on the left side. I'm aware of this and will try to fix it eventually

![2023-09-10_10h40_43](https://github.com/KirisameMarisa/BocchiTracker/assets/5306954/cf7cf1cd-5226-43d4-856a-e3d3d55020c5)

**Comment by KirisameMarisa** [comment]
I'll take over.
I will refer to your repository! Thanks!
@Web-NAVI

**Comment by Web-NAVI** [comment]
Okay, you might take a look at the "Scetch" page, that's where I was testing if the original code worked on Blazor. Also be aware that Blazor won't like reactive properies and I recall having many issues with them.

Also it might be a bit messy since it wasn't intended to become the final port so maybe it will be better to have it just as a reference rather than try to pick it up. 
![image](https://github.com/KirisameMarisa/BocchiTracker/assets/98480588/fdcb5dcd-4a28-4387-a767-8593e6a31c53)

**Comment by KirisameMarisa** [comment]
long time no see!

I recently got a regular job again, so it will be longer than spring before I start on the task.
If you still have some desire to raise Bocchi-chan, I'd be happy to return the task assignment to you, what do you think?

@Web-NAVI

**Comment by Web-NAVI** [comment]
Of course! Also now I have more time than before so if you still need help I could start with this.

**Comment by KirisameMarisa** [comment]
yeaay!! thank you very much!
I will trun this assigment back over to you!

Ah! If you want bocchi-chan icon, I draw her
I drew this file myself!
https://github.com/KirisameMarisa/BocchiTracker/blob/master/Plugins/Godot/project/icon.png

**Comment by Web-NAVI** [comment]
I guess you saw me trying to make an icon with my limited Photoshop skills😅.

I will gladly have you draw an icon! Really nice drawing btw.

Also, I took a look at how to continue.

My plan is to keep doing it with Blazor (html + C#) on top of MAUI (to make it multiplatform).
 Unless you want a pure MAUI (XAML), Even tho my XAML skills are quite limited so that may take longer.

**Comment by KirisameMarisa** [comment]
> I will gladly have you draw an icon! Really nice drawing btw.

Thank you! Of course! Leave it to me!

Yeah! I hope you continue to develop Blazor (html + C#) on top of MAUI!

I will continue to develop WPF (or MAUI) as a powerful Windows client!
(I haven't decided yet whether to unify with Blazor or not)
